be/test/storage/size_tiered_compaction_policy_test.cpp

```diff
@@ -52,17 +52,18 @@ class SizeTieredCompactionPolicyTest : public testing::Test {
     }
 
     void rowset_writer_add_rows(std::unique_ptr<RowsetWriter>& writer, int64_t level) {
+        static int64_t id = 0;
```

Comment:
It will be better to be atomic<int64_t> in case of executing concurrently.

Reply:
There is no need, as the case does not rely on specific values; it is ok the values within one case are different.